src/index.jsx

```diff
@@ -146,6 +147,14 @@ var MaskedInput = React.createClass({
       setTimeout(this._updateInputSelection, 0)
       this.props.onChange(e)
     }
+    else {
```

Comment:
Could you educate me on why the `if (this.mask.paste(e.clipboardData.getData('Text')) {...}` part just before this block returns falsy and falls through to this?